login/core/validations/user_validation/DbValidation.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+
+const SequelizeOperators = Sequelize.Op;
+const UserModel = require('../../../db/models/user_models/UserModel.js');
+const createUserAccount = require('../../../routers/controllers/user_controllers/UserServices.js');
```

Comment:
rename the variable to userService from createUserAccount, think in terms of objects, it makes sense that a userService creates an user account(maybe at this point rename the function to create as well? It's obvious that a userService is not creating bananas)